vendor/modules.txt

```diff
@@ -196,6 +196,8 @@ github.com/gophercloud/gophercloud/openstack/identity/v2/tokens
 github.com/gophercloud/gophercloud/openstack/identity/v3/tokens
 github.com/gophercloud/gophercloud/openstack/utils
 github.com/gophercloud/gophercloud/pagination
+# github.com/gorilla/websocket v1.4.1
+github.com/gorilla/websocket
```

Comment:
I presume this is an indirect dependency that we're not actually using?

Reply:
Yes.

I'm planning to do some refactoring in the language server, that will stop pulling in this indirect dependency.

Reply:
Fixed in https://github.com/prometheus-community/promql-langserver/pull/116 .